src/libPMacc/include/simulationControl/SimulationHelper.hpp

```diff
@@ -198,80 +213,89 @@ class SimulationHelper : public IPlugin
      */
     void startSimulation()
     {
-        uint32_t currentStep = init();
-        Environment<>::get().SimulationDescription().setCurrentStep( currentStep );
+        init();
 
-        tInit.toggleEnd();
-        if (output)
+        for (uint32_t softRestartLoop = 0; softRestartLoop <= softRestarts; ++softRestartLoop)
```

Comment:
Could you pls provide some more meaningful name for the loop index e.g. nthSoftRestart or softRestart_i